idrstream/cellprofiler_metadata.py

```diff
@@ -0,0 +1,20 @@
+"""
+This function will convert the `data_to_process.tsv` file into a .csv so that CellProfiler can read in the metadata and connect the appropriate
+metadata to the images from each batch. 
+"""
+import pandas as pd
+
+def convert_tsv_to_csv(data_to_process_tsv: str):
```

Comment:
I think you meant for `data_to_process_tsv` to be a `pathlib.Path`, correct? (if so make sure to import pathlib as well!)